[2024-10-12T13:03:58.380+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-10-12T13:03:58.410+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Taxi_trip_pipeline.dbt_transformation_layer.dbt_run_model_zone_lookup manual__2024-10-12T13:03:08.986267+00:00 [queued]>
[2024-10-12T13:03:58.426+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Taxi_trip_pipeline.dbt_transformation_layer.dbt_run_model_zone_lookup manual__2024-10-12T13:03:08.986267+00:00 [queued]>
[2024-10-12T13:03:58.427+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-10-12T13:03:58.443+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): dbt_transformation_layer.dbt_run_model_zone_lookup> on 2024-10-12 13:03:08.986267+00:00
[2024-10-12T13:03:58.457+0000] {standard_task_runner.py:63} INFO - Started process 156 to run task
[2024-10-12T13:03:58.460+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Taxi_trip_pipeline', 'dbt_transformation_layer.dbt_run_model_zone_lookup', 'manual__2024-10-12T13:03:08.986267+00:00', '--job-id', '12', '--raw', '--subdir', 'DAGS_FOLDER/taxi_trip.py', '--cfg-path', '/tmp/tmps1y_rl17']
[2024-10-12T13:03:58.463+0000] {standard_task_runner.py:91} INFO - Job 12: Subtask dbt_transformation_layer.dbt_run_model_zone_lookup
[2024-10-12T13:03:58.525+0000] {task_command.py:426} INFO - Running <TaskInstance: Taxi_trip_pipeline.dbt_transformation_layer.dbt_run_model_zone_lookup manual__2024-10-12T13:03:08.986267+00:00 [running]> on host ebdcf85807fc
[2024-10-12T13:03:58.624+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Mahmoud' AIRFLOW_CTX_DAG_ID='Taxi_trip_pipeline' AIRFLOW_CTX_TASK_ID='dbt_transformation_layer.dbt_run_model_zone_lookup' AIRFLOW_CTX_EXECUTION_DATE='2024-10-12T13:03:08.986267+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-12T13:03:08.986267+00:00'
[2024-10-12T13:03:58.625+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-10-12T13:03:58.642+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-10-12T13:03:58.643+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'dbt run --profiles-dir /opt/***/DBT_Project  --project-dir /opt/***/dbt_trip --models zone_lookup']
[2024-10-12T13:03:58.665+0000] {subprocess.py:86} INFO - Output:
[2024-10-12T13:04:02.157+0000] {subprocess.py:93} INFO - [0m13:04:02  Running with dbt=1.8.0
[2024-10-12T13:04:02.605+0000] {subprocess.py:93} INFO - [0m13:04:02  Registered adapter: postgres=1.8.0
[2024-10-12T13:04:02.899+0000] {subprocess.py:93} INFO - [0m13:04:02  Unable to do partial parsing because profile has changed
[2024-10-12T13:04:05.479+0000] {subprocess.py:93} INFO - [0m13:04:05  [[33mWARNING[0m]: Deprecated functionality
[2024-10-12T13:04:05.479+0000] {subprocess.py:93} INFO - The `tests` config has been renamed to `data_tests`. Please see
[2024-10-12T13:04:05.480+0000] {subprocess.py:93} INFO - https://docs.getdbt.com/docs/build/data-tests#new-data_tests-syntax for more
[2024-10-12T13:04:05.480+0000] {subprocess.py:93} INFO - information.
[2024-10-12T13:04:05.941+0000] {subprocess.py:93} INFO - [0m13:04:05  [[33mWARNING[0m]: Found patch for macro "payment_type_mapping" which was not found
[2024-10-12T13:04:05.943+0000] {subprocess.py:93} INFO - [0m13:04:05  [[33mWARNING[0m]: Found patch for macro "rate_code_mapping" which was not found
[2024-10-12T13:04:05.946+0000] {subprocess.py:93} INFO - [0m13:04:05  [[33mWARNING[0m]: Found patch for macro "store_and_fwd_flag_mapping" which was not found
[2024-10-12T13:04:05.949+0000] {subprocess.py:93} INFO - [0m13:04:05  [[33mWARNING[0m]: Found patch for macro "trip_type_mapping" which was not found
[2024-10-12T13:04:05.952+0000] {subprocess.py:93} INFO - [0m13:04:05  [[33mWARNING[0m]: Found patch for macro "vendor_mapping" which was not found
[2024-10-12T13:04:06.104+0000] {subprocess.py:93} INFO - [0m13:04:06  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2024-10-12T13:04:06.105+0000] {subprocess.py:93} INFO - There are 2 unused configuration paths:
[2024-10-12T13:04:06.105+0000] {subprocess.py:93} INFO - - models.fact_trip.Dimensions
[2024-10-12T13:04:06.105+0000] {subprocess.py:93} INFO - - models.fact_trip.staging
[2024-10-12T13:04:06.430+0000] {subprocess.py:93} INFO - [0m13:04:06  Found 4 models, 6 data tests, 3 sources, 566 macros
[2024-10-12T13:04:06.434+0000] {subprocess.py:93} INFO - [0m13:04:06
[2024-10-12T13:04:06.636+0000] {subprocess.py:93} INFO - [0m13:04:06  Concurrency: 3 threads (target='dev')
[2024-10-12T13:04:06.637+0000] {subprocess.py:93} INFO - [0m13:04:06
[2024-10-12T13:04:06.642+0000] {subprocess.py:93} INFO - [0m13:04:06  1 of 1 START sql view model analysis.zone_lookup ............................... [RUN]
[2024-10-12T13:04:06.856+0000] {subprocess.py:93} INFO - [0m13:04:06  1 of 1 OK created sql view model analysis.zone_lookup .......................... [[32mCREATE VIEW[0m in 0.21s]
[2024-10-12T13:04:06.875+0000] {subprocess.py:93} INFO - [0m13:04:06
[2024-10-12T13:04:06.875+0000] {subprocess.py:93} INFO - [0m13:04:06  Finished running 1 view model in 0 hours 0 minutes and 0.44 seconds (0.44s).
[2024-10-12T13:04:06.978+0000] {subprocess.py:93} INFO - [0m13:04:06
[2024-10-12T13:04:06.980+0000] {subprocess.py:93} INFO - [0m13:04:06  [32mCompleted successfully[0m
[2024-10-12T13:04:06.981+0000] {subprocess.py:93} INFO - [0m13:04:06
[2024-10-12T13:04:06.982+0000] {subprocess.py:93} INFO - [0m13:04:06  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2024-10-12T13:04:08.182+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-10-12T13:04:08.183+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-10-12T13:04:08.214+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=Taxi_trip_pipeline, task_id=dbt_transformation_layer.dbt_run_model_zone_lookup, run_id=manual__2024-10-12T13:03:08.986267+00:00, execution_date=20241012T130308, start_date=20241012T130358, end_date=20241012T130408
[2024-10-12T13:04:08.242+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-10-12T13:04:08.267+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-12T13:04:08.268+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
